CHANGELOG.md

```diff
@@ -101,6 +101,11 @@ Adding a new version? You'll need three changes:
   [#5128](https://github.com/Kong/kubernetes-ingress-controller/pull/5128)
 - Added `-init-cache-sync-duration` CLI flag. This flag configures how long the controller waits for Kubernetes resources to populate at startup before generating the initial Kong configuration. It also fixes a bug that removed the default 5 second wait period.
   [#5238](https://github.com/Kong/kubernetes-ingress-controller/pull/5238)
+- Added support for `HTTPRouteTimeoutBackendRequest` in Gateway API.
+  Due to only one field being available in the Gateway API to control this behavior,
+  when users set `spec.rules[].timeouts` in HTTPRoute,
+  KIC will also set ReadTimeout, WriteTimeout and ConnectTimeout for the service to this value.
```

Comment:
```suggestion
  KIC will set ReadTimeout, WriteTimeout and ConnectTimeout in the Kong service to this value.
```
How about using `read_timeout`, `write_timeout` and `connect_timeout` format to be consistent with Kong's admin APIs?

Reply:
Changed. Thanks